app/tools/android/customize.py

```diff
@@ -331,12 +331,18 @@ def CustomizeJava(app_info, app_url, app_local_path, keep_screen_on):
   dest_activity = os.path.join(app_pkg_dir, name + 'Activity.java')
   ReplaceString(dest_activity, 'org.xwalk.app.template', package)
   ReplaceString(dest_activity, 'AppTemplate', name)
-  manifest_file = os.path.join(app_dir, 'assets', 'www', 'manifest.json')
-  if os.path.isfile(manifest_file):
-    ReplaceString(
-        dest_activity,
-        'loadAppFromUrl("file:///android_asset/www/index.html")',
-        'loadAppFromManifest("file:///android_asset/www/manifest.json")')
+  if manifest:
+    manifest_name = manifest.input_path.split(os.path.sep)[-1]
+    manifest_file = os.path.join(app_dir, 'assets', 'www', manifest_name)
+    if os.path.isfile(manifest_file):
```

Comment:
Yeah, but can't you assume this code is always called after everything has been copied to `assets/www` (meaning that if copying has failed the entire program should have failed and exited)? I'd like to avoid having duplicated checks everywhere like this one.